Gems/Atom/Tools/AtomToolsFramework/Code/Source/Inspector/PropertyWidgets/PropertyStringBrowseEditCtrl.cpp

```diff
@@ -64,7 +64,7 @@ namespace AtomToolsFramework
     {
         if (attrib == AZ_CRC_CE("LineEditReadOnly"))
         {
-            if (bool value = true; attrValue->Read<bool>(value))
+            if (bool value = true; attrValue->Read<decltype(value)>(value))
```

Comment:
Do you even need the template specification? I think it can infer the type.
```suggestion
            if (bool value = true; attrValue->Read(value))
```

Reply:
It may not be required. I just changed the model to be consistent. I've been bitten by that before after submitting changes to AR for different platforms. Better safe than sorry.